### PR TITLE
Start documenting worker internals

### PR DESCRIPTION
**Pull request by jpopelka**

### CODE REVIEW

File: f8a_worker/utils.py [hunk 1]
```diff
@@ -248,14 +257,13 @@ def target(**kwargs):
 
     @staticmethod
     def get_command_output(args, graceful=True, is_json=False, timeout=300, **kwargs):
-        """Wrapper around get_command_output() with implicit timeout of 5 minutes."""
+        """Wrapp get_command_output() with implicit timeout of 5 minutes."""
```
Comment (tisnik) [inline f8a_worker/utils.py]: Wrap

**Comment by tisnik** [comment]
Very nice @jpopelka. Thanks a lot! LGTM (just one nitpick ^^)

**Comment by jpopelka** [comment]
[test]

**Comment by tisnik** [comment]
LGTM!

**Pull request merged**